.larasail/database

```diff
@@ -36,7 +36,7 @@ create_database_details() {
     echo "[client]" >> "/etc/.larasail/tmp/mysql.cnf"
     echo "password=$ROOT_PASSWORD" >> "/etc/.larasail/tmp/mysql.cnf"
 
-    echo "$DATABASE_NAME:$DATABASE_PASSWORD" > "/etc/.larasail/tmp/$DATABASE_USER.dbpass"
+    echo "$DATABASE_PASSWORD" > "/etc/.larasail/tmp/$DATABASE_USER.$DATABASE_NAME.dbpass"
```

Comment:
I have taken the opportunity to tweak `dbpass` file name format while this PR is still open.
This format is more intuitive when looking at the file name and follows `namespace.database` convention.